torch/utils/__init__.py

```diff
@@ -4,7 +4,7 @@
 from .throughput_benchmark import ThroughputBenchmark
 from ._crash_handler import enable_minidumps, disable_minidumps, enable_minidumps_on_exceptions
 from .cpp_backtrace import get_cpp_backtrace
-from .backend_registration import rename_privateuse1_backend
+from .custom_backend_registration import rename_privateuse1_backend, get_amp_supported_dtype, set_amp_supported_dtype
```

Comment:
Is there a reason for the file name change? Unfortunately, we probably don't want to risk changing the module since it's technically BC breaking (although I'm not sure how heavily used `rename_privateuse1_backend()` is yet).

Reply:
haaa,  as for custom backend, we would register other funcs for custom backend in the future. And as what you said, now i think just keep it as it is.